package.json

```diff
@@ -4,9 +4,10 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "compile": "tsc && prettier ./src/**/*.js --write",
```

Comment:
I'm pretty sure there's a way to have `vue-cli-service` understand typescript directly. We should look into this instead of having a separate compile step.

https://cli.vuejs.org/core-plugins/typescript